### PR TITLE
Feat/backskeleton

### PR DESCRIPTION
**Pull request by tchan247**

### CODE REVIEW

**Comment by ardsouza** [comment]
please fix CONTRIBUTING.md

**Comment by ardsouza** [comment]
lgtm

File: server/db/models/repo.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+var db = require('../config');
+
+var Repo = new db.Model.extend({
+  tableName: 'Repo'
```
Comment (dxuehu) [inline server/db/models/repo.js]: relationships missing and probably other stuff

File: server/server.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+var express = require('express');
+var app = express();
+var db = require('./db/config.js');
+// var Repos = require('./db/collections/repos.js');
+// var Commits = require('./db/collections/commits.js');
+// var Repo = require('./db/models/repo.js');
+// var Commit = require('./db/models/commit.js');
+
+app.get('/', function(){
```
Comment (dxuehu) [inline server/server.js]: should serve static files

**Pull request merged**